upload/admin/language/nl-nl/extension/payment/emspay_klarnapaynow.php

```diff
@@ -48,7 +48,7 @@
 $_['info_help_api_key'] = 'Kopieer uw EMS Online API key van uw merchant portal.';
 $_['info_help_total'] = 'De betaalmethode wordt alleen getoond als het totaalbedrag van een bestelling hoger is dan dit bedrag.';
 $_['info_plugin_not_configured'] = 'EMS Online: Klarna Pay Now plugin is niet ingesteld.';
-$_['info_help_klarna_ip_filter'] = 'U  kunt specifieke IP adressen opgeven waarvoor de betaalmethode zichtbaar is, bv als u wilt testen  (bijvoorbeeld: 128.0.0.1, 255.255.255.255). Vult u niets in dan is de betaalmethode voor alle IP adressen zichtbaar.';
-$_['entry_klarna_ip_filter'] = 'IP-Filteren:';
-$_['entry_klarna_test_api_key'] = 'Test API Key:';
-$_['info_help_klarna_test_api_key'] = 'Vul hier de API Key in van uw test webshop.';
\ No newline at end of file
+$_['info_help_klarnapaylater_ip_filter'] = 'U  kunt specifieke IP adressen opgeven waarvoor de betaalmethode zichtbaar is, bv als u wilt testen  (bijvoorbeeld: 128.0.0.1, 255.255.255.255). Vult u niets in dan is de betaalmethode voor alle IP adressen zichtbaar.';
```

Comment:
Why are there fields relating to klarna pay later?

Reply:
Fixed(deleted)